exporter/exporter_test.go

```diff
@@ -0,0 +1,88 @@
+package exporter
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/lightstep/terraform-provider-lightstep/client"
+)
+
+func TestExportToHCL(t *testing.T) {
+	export := func(d *client.UnifiedDashboard) (string, error) {
+		var buf bytes.Buffer
+		err := exportToHCL(&buf, d)
+		return buf.String(), err
+	}
+
+	testCases := []struct {
+		QueryString string
+		Expected    string
+	}{
+		{
+			QueryString: "",
+			Expected:    "",
+		},
+		{
+			QueryString: "metric requests | rate 10m",
+			Expected:    "query_string        = metric requests | rate 10m",
```

Comment:
this is the kind of thing i mentioned about -- do we know terraform won't complain about the lack of quotes here?